.jshintrc

```diff
@@ -0,0 +1,16 @@
+{
+	"boss": true,
+	"curly": true,
+	"eqeqeq": true,
+	"eqnull": true,
+	"expr": true,
+	"immed": true,
+	"noarg": true,
+	"onevar": true,
```

Comment:
we don't really do onevar anymore. https://github.com/jquery/contribute.jquery.org/commit/a1fbe20dd1d0082490289c2ef411860cc98522dd